packages/presentation/lib/screen/splash/splash_bloc.dart

```diff
@@ -24,7 +27,8 @@ class SplashBlocImpl extends BlocImpl implements SplashBloc {
     await _blocDelayUseCase();
     appNavigator.popAndPush(
       HomeScreen.page(
-        HomeScreenArguments(),
+        HomeScreenArguments(
+            anticipatedMoviesResponse: [], trendingMoviesResponse: []),
       ),
```

Comment:
why do u pass an empty arguments? and then in bloc u updating ui with empty data. It force to rebuild you ui, it have no sense